src/main/java/jenkins/security/plugins/ldap/FromGroupSearchLDAPGroupMembershipStrategy.java

```diff
@@ -68,6 +81,13 @@ public void setAuthoritiesPopulator(LdapAuthoritiesPopulator authoritiesPopulato
         return getAuthoritiesPopulator().getGrantedAuthorities(ldapUser);
     }
 
+    @Override
+    public Set<String> getGroupMembers(String groupDn, LDAPConfiguration conf) throws DataAccessException {
+        LDAPExtendedTemplate template = conf.getLdapTemplate();
+        String[] memberAttributes = { "member", "uniqueMember", "memberUid" };
```

Comment:
(Resurrecting outdated comment). I think this covers the majority of member attributes on groups. Thoughts on whether the hardcoding here is ok or if this should be configurable by the user?